src/main/java/org/jenkinsci/plugins/codescene/CodeSceneBuilder.java

```diff
@@ -314,6 +320,21 @@ private CommitRange getRangeFromPreviousSuccessfulCommit(EnvVars env, Commit cur
         }
     }
 
+    private EnvVars getEnvVar(FlowExecution execution) throws ExecutionException, InterruptedException, IOException {
+        EnvVars envVars = new EnvVars();
+        if(execution != null) {
+            ListenableFuture<List<StepExecution>> executions = execution.getCurrentExecutions(true);
```

Comment:
What's the idea behind `true` parameter? Can you link the source that inspired this implementation?

Reply:
https://javadoc.jenkins.io/plugin/workflow-api/org/jenkinsci/plugins/workflow/flow/FlowExecution.html